src/views/blocks/tickets/submit-button.php

```diff
@@ -53,5 +53,5 @@
 	<?php endif; ?>
```

Comment:
This view also needs @version TBD and @since TBD Use tribe_get_ticket_label_plural() for "tickets" string.